### PR TITLE
reviews consensus docs

### PR DESCRIPTION
**Pull request by Maria-Nosyk**

### CODE REVIEW

File: src/advanced/consensus/requests.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ has the following fields:
   Each bit of this field indicates the need to send a `Prevote` message from
   the corresponding validator (if the bit value is 1, `Prevote` is requested;
   otherwise, `Prevote` is not needed). Indexing of the `validator` bits
-  corresponds to indexing the validator public keys in the
+  corresponds to indexing of the validator public keys in the
```
Comment (MartaDubno) [inline src/advanced/consensus/requests.md]: I'd suggest: 'corresponds to the indexing of validator public keys'

File: src/advanced/consensus/requests.md [hunk 2]
```diff
@@ -240,7 +241,7 @@ Cancel all requests.
 
 ## Requests Processing
 
-This algorithm determines the processing of different types of request messages
+This algorithm determines processing of different types of request messages
```
Comment (MartaDubno) [inline src/advanced/consensus/requests.md]: I'd perhaps leave the definite article here.

File: src/advanced/consensus/specification.md [hunk 3]
```diff
@@ -254,12 +257,13 @@ round are placed in a separate queue (`queued`).
...
-  hash map (i.e., the message has been processed previously), ignore the message.
+  hash map (i.e., the message has been processed previously), ignore the
+  message.
 - Check `propose.prev_hash` correctness.
 - Check that the specified validator is the leader for the given round.
 - Check that the proposal does not contain previously committed transactions
-  (`Propose` messages contain only hashes of transactions, so the absence of
-  hashes in the table of committed transactions is checked).
+  (`Propose` messages contain only hashes of transactions, so absence of
+  hashes in the table of the committed transactions is checked).
```
Comment (MartaDubno) [inline src/advanced/consensus/specification.md]: I'd perhaps remove the definite article before 'committed transactions'

File: src/advanced/consensus/specification.md [hunk 4]
```diff
@@ -314,23 +319,24 @@ round are placed in a separate queue (`queued`).
...
     blockchain height.
 
-- Check the block message:
+- Check the `Block` message:
 
     - The key in the `to` field must match the key of the node.
     - `block.prev_hash` must match the hash of the latest committed block.
     - The block height must be equal to the current height of the node.
-    - The number of `Precommit` messages from different validators
+    - The number of `precommit` messages from different validators
```
Comment (slowli) [inline src/advanced/consensus/specification.md]: Unnecessary change.

File: src/advanced/consensus/specification.md [hunk 5]
```diff
@@ -314,23 +319,24 @@ round are placed in a separate queue (`queued`).
...
       must be sufficient to reach consensus.
-    - All `Precommit` messages must be correct.
+    - All `precommit` messages must be correct.
```
Comment (slowli) [inline src/advanced/consensus/specification.md]: Unnecessary change.

File: src/advanced/consensus/specification.md [hunk 6]
```diff
@@ -368,33 +374,36 @@ round are placed in a separate queue (`queued`).
...
-  which the proposal belongs.
+- If the node does not have a saved PoL, send a `Prevote` message in the round
+  to which the proposal belongs.
 - For each round `r` in the interval
   `[max(locked_round + 1, propose.round), current_round]`:
 
-    - If the node has +2/3 `Prevote`s for `propose` in `r`, then
-    proceed to [Availability of +2/3 Prevotes](#availability-of-23-prevotes) for
-    `propose` in `r`.
+    - If the node has +2/3 `Prevote`s for `Propose` in `r`, then
```
Comment (slowli) [inline src/advanced/consensus/specification.md]: Unnecessary change.

File: src/advanced/consensus/specification.md [hunk 7]
```diff
@@ -368,33 +374,36 @@ round are placed in a separate queue (`queued`).
...
+    proceed to [Availability of +2/3 `Prevote`s](#availability-of-23-prevotes)
+    for `Propose` in `r`.
```
Comment (slowli) [inline src/advanced/consensus/specification.md]: Unnecessary change.

File: src/advanced/consensus/specification.md [hunk 8]
```diff
@@ -368,33 +374,36 @@ round are placed in a separate queue (`queued`).
...
 
 - For each round `r` in the interval `[propose.round, current_round]`:
 
-    - If +2/3 `Precommit`s are available for `propose` in `r` and with
+    - If +2/3 `Precommit`s are available for `Propose` in `r` and with
```
Comment (slowli) [inline src/advanced/consensus/specification.md]: Unnecessary change.

File: src/advanced/consensus/specification.md [hunk 9]
```diff
@@ -368,33 +374,36 @@ round are placed in a separate queue (`queued`).
...
       the same `state_hash`, then:
 
         - Execute the proposal, if it has not yet been executed.
-        - Check that the node’s `state_hash` after applying transactions in `propose`
-          coincides with the `state_hash` in the aforementioned +2/3 `Precommit`s.
+        - Check that the node’s `state_hash` after applying transactions in
+          `Propose`
```
Comment (slowli) [inline src/advanced/consensus/specification.md]: Unnecessary change.

File: src/advanced/consensus/specification.md [hunk 10]
```diff
@@ -291,10 +296,10 @@ round are placed in a separate queue (`queued`).
   round with the given `state_hash`.
 - If:
 
-    - the node has formed +2/3 `Precommit`s for the same round, `propose_hash`
+    - the node has formed +2/3 `precommit`s for the same round, `propose_hash`
```
Comment (slowli) [inline src/advanced/consensus/specification.md]: Unnecessary change.

File: src/advanced/consensus/specification.md [hunk 11]
```diff
@@ -273,12 +277,13 @@ round are placed in a separate queue (`queued`).
   in `prevote.round`.
 - If:
 
-    - the node has formed +2/3 `Prevote` messages for the same round and `propose_hash`
+    - the node has formed +2/3 `prevote` messages for the same round and
+    `propose_hash`
     - `locked_round < prevote.round`
-    - the node knows a `Propose` message referenced by this `prevote`
+    - the node knows a `propose` message referenced by this `prevote`
```
Comment (slowli) [inline src/advanced/consensus/specification.md]: Unnecessary change.

File: src/advanced/consensus/specification.md [hunk 12]
```diff
@@ -273,12 +277,13 @@ round are placed in a separate queue (`queued`).
   in `prevote.round`.
 - If:
 
-    - the node has formed +2/3 `Prevote` messages for the same round and `propose_hash`
+    - the node has formed +2/3 `prevote` messages for the same round and
```
Comment (slowli) [inline src/advanced/consensus/specification.md]: Unnecessary change.

File: src/advanced/consensus/specification.md [hunk 13]
```diff
@@ -213,12 +216,12 @@ round are placed in a separate queue (`queued`).
 - If the transaction is already in the pool of unconfirmed transactions,
   ignore it.
 - Add the transaction to the pool of unconfirmed transactions.
-- For every known proposal `propose` in which this transaction is included:
+- For every known proposal `propose` where this transaction is included:
 
     - Exclude the hash of this transaction from the list of unknown transactions
-      for the `propose`.
+      for the `Propose`.
```
Comment (slowli) [inline src/advanced/consensus/specification.md]: Unnecessary change.

File: src/advanced/consensus/specification.md [hunk 14]
```diff
@@ -213,12 +216,12 @@ round are placed in a separate queue (`queued`).
...
     - If the number of unknown transactions for the proposal becomes zero,
-      proceed to [Full proposal](#full-proposal) state for `propose`.
+      proceed to [Full proposal](#full-proposal) state for `Propose`.
```
Comment (slowli) [inline src/advanced/consensus/specification.md]: Unnecessary change.
